newdemojail.yaml

```
@@ -0,0 +1,772 @@
+entityDefinitions:
+  inmate:
+    fqn: "general.person"
+    entitySetName: "DemoPerson"
+    propertyDefinitions:
+      nc.SubjectIdentification:
+        type: "nc.SubjectIdentification"
+        column: "SubjectIdentification"
+      nc.PersonSurName:
+        type: "nc.PersonSurName"
+        column: "LastName"
+      nc.PersonGivenName:
+        type: "nc.PersonGivenName"
+        column: "FirstName"
+      nc.PersonMiddleName:
+        type: "nc.PersonMiddleName"
+        column: "MiddleName"
+      nc.PersonBirthDate:
+        type: "nc.PersonBirthDate"
+        column: "BirthDate"
+      nc.PersonSex:
+        type: "nc.PersonSex"
+        column: "Sex"
+      nc.SSN:
+        type: "nc.SSN"
+        column: "SocialSecurityNumber"
+      nc.PersonRace:
+        type: "nc.PersonRace"
+        column: "Race"
+      nc.PersonEthnicity:
+        type: "nc.PersonEthnicity"
+        column: "Ethnicity"
+    name: "inmate"
+  persondetails:
+    fqn: "ol.persondetails"
+    entitySetName: "DemoPersondetails"
+    propertyDefinitions:
+      general.id:
+        type: "general.id"
+        column: "SubjectIdentification"
+      person.maritalstatus:
+        type: "person.maritalstatus"
+        column: "MaritalStatus"
+      person.gender:
+        type: "person.gender"
+        column: "Gender"
+      person.juvenile:
+        type: "person.juvenile"
+        column: "Juvenile"
+    name: "persondetails"
+  education:
+    fqn: "ol.education"
+    entitySetName: "DemoEducation"
```

```suggestion
    entitySetName: "DemoInmateEducation"
```
Here's another existing one. I'm not going to keep checking for the rest but **please do** .

---

looks like this change dod not get saved.